.changeset/flat-roses-happen.md

```diff
@@ -0,0 +1,13 @@
+---
+'@backstage/plugin-scaffolder-backend-module-bitbucket-server': minor
+'@backstage/plugin-scaffolder-backend-module-bitbucket-cloud': minor
+'@backstage/plugin-scaffolder-backend-module-bitbucket': minor
```

Comment:
Wonder if we should do a special changeset just for the `-bitbucket` one, so that we can throw in some migration docs here with some deprecation notes?

Reply:
Sure, I can do that.

Reply:
I have created separate changesets.